umh.docs.umh.app/content/en/docs/production-guide/administration/reduce-database-size.md

```diff
@@ -44,31 +44,92 @@ Connect to the `factoryinsight` database:
+```
+
 ## Enable data compression
 
-To enable data compression, you need to execute the following SQL command from
-the database shell:
+You can find sample SQL commands to enable data compression here:
 
-```sql
-SELECT add_retention_policy('processvaluetable', INTERVAL '7 days');
-```
+### Sample command for the processvaluetable in the factoryinsight database:
+
+1. First, turn on compression:
+
+    ```sql
+    ALTER TABLE processvaluetable SET (timescaledb.compress, timescaledb.compress_segmentby = 'asset_id', timescaledb.compress_orderby = 'valuename');
+    ```
+    This command set `asset_id` as the key for the compressed segments 
+    and orders the table by `valuename`.
+
+2. Then, create a compression policy:
+
+    ```sql
+    SELECT add_compression_policy('processvaluetable', INTERVAL '7 days');
+    ```
+
+    This command will set a compression policy on the `processvaluetable` table,
+    which will compress data older than 7 days.
+
+### Sample command for the tag table in the umh_v2 database:
+
+1. First, turn on compression:
+
+    ```sql
+    ALTER TABLE tag SET (timescaledb.compress, timescaledb.compress_segmentby = 'asset_id', timescaledb.compress_orderby = 'name');
+    ```
+    This command set `asset_id` as the key for the compressed segments 
+    and orders the table by `name`.
+
+2. Then, create a compression policy:
+
+    ```sql
+    SELECT add_compression_policy('tag', INTERVAL '2 weeks');
+    ```
+
+    This command will set a compression policy on the `tag` table,
+    which will compress data older than 2 weeks.
+
+
+Refer to [the official documentation](https://docs.timescale.com/api/latest/compression/alter_table_compression/)
+for more detailed information about these queries.
 
-This command will set a retention policy on the `processvaluetable` table, which
-will delete data older than 7 days.
 
 ## Enable data retention
 
-To enable data retention, you need to execute the following SQL command from the
-database shell:
+You can find sample SQL commands to enable data retention here:
 
-```sql
-SELECT add_compression_policy('processvaluetable', INTERVAL '7 days');
-```
+### Sample command for the processvaluetable in the factoryinsight database:
 
-This command will set a compression policy on the `processvaluetable` table,
-which will compress data older than 7 days.
+  ```sql
+  SELECT add_retention_policy('processvaluetable', INTERVAL '7 days');
+  ```
+
+  This command will set a retention policy on the `processvaluetable` table, which
+  will delete data older than 7 days.
+
+### Sample command for the tag table in the umh_v2 database:
+
+  ```sql
+  SELECT add_retention_policy('tag', INTERVAL '3 months');
+  ```
+
+  This command will set a retention policy on the `tag` table, which
+  will delete data older than 3 months.
+
+Refer to [the official documentation](https://docs.timescale.com/api/latest/data-retention/add_retention_policy/)
+for more detailed information about these queries.
```

Comment:
For the data retention paragraph, I would write something like this:

``````markdown
## Enable data retention

Enabling data retention consists in only adding the policy with the desired retention interval. Refer to [the official documentation](https://docs.timescale.com/api/latest/data-retention/add_retention_policy/) for more detailed information about these queries.

```sql
SELECT add_retention_policy('<target-table>', INTERVAL '<time>');
```

``````

Reply:
updated